### PR TITLE
Pattern Library Grid doesn't look good in large browser windows

### PR DESCRIPTION
**Pull request by synbot**
## Description
The Pattern Library's Grid doesn't look good in large browser sizes. It seems to render better when I shrink the window down to medium / small sizes, but when my browser is in full-screen, this is what I see:

screen shot 2015-02-24 at 12 40 54 pm

## Details
http://localhost:9000/pattern-library/Grid

### CODE REVIEW

File: application/ui/scss/pattern-library/_pl-layout.scss [hunk 1]
```diff
@@ -53,32 +53,25 @@ $pl-header-height : rem-calc(64);
...
+    &.example .row .column,
+    &.example .row .columns {
+        font-size   : 11px;
+        background  : #f4f4f4;
+        height      : 60px;
+        line-height : 60px;
+    }
+    &.example .row{
+        margin-bottom : 20px;
+    }
```
Comment (natestormer) [inline application/ui/scss/pattern-library/_pl-layout.scss]: add line breaks before and after nested blocks

**Comment by natestormer** [comment]
:interrobang:

**Comment by kylecesmat** [comment]
:star2:

**Comment by natestormer** [comment]
:pray:

**Comment by aecarr2** [comment]
:hand:

**Comment by aecarr2** [comment]
:+1:

**Pull request merged**